packages/dev/core/src/Shaders/ShadersInclude/helperFunctions.fx

```diff
@@ -45,34 +45,94 @@ mat3 inverseMat3(mat3 inMatrix) {
               b21, (-a21 * a00 + a01 * a20), (a11 * a00 - a01 * a10)) / det;
 }
 
+#if USEEXACTSRGBCONVERSIONS
+vec3 toLinearSpaceExact(vec3 color)

```

Comment:
Maybe you can add a comment pointing to the reference for these computation (https://en.wikipedia.org/wiki/SRGB)?